pkg/scheduler/plugins/numaaware/numaaware.go

```diff
@@ -136,7 +137,7 @@ func (pp *numaPlugin) OnSessionOpen(ssn *framework.Session) {
 
 			klog.V(4).Infof("[numaaware] hits for task %s container '%v': %v on node %s, besthit: %v",
 				task.Name, container.Name, providersHints, node.Name, hit)
-			resAssignMap := policy.Allocate(&container, &hit, node.NumaSchedulerInfo, resNumaSets, pp.hintProviders)
+			resAssignMap := policy.Allocate(&con, &hit, node.NumaSchedulerInfo, resNumaSets, pp.hintProviders)
```

Comment:
So why new thest variables?

Reply:
for-range is not recommended reference value directly, so new variable to fix lint checks.

Reply:
Well, thanks for the advice. Maybe we can pick some other bug fix which is more valuable next time.